core/captin.go

```diff
@@ -40,6 +40,7 @@ func NewCaptin(configMap interfaces.ConfigMapperInterface) *Captin {
 	store := stores.NewMemoryStore()
 	senderMapping := map[string]interfaces.EventSenderInterface{
 		"http": &senders.HTTPEventSender{},
+		"beanstalkd": &senders.BeanstalkdSender{},
```

Comment:
i think we don't have to add `beanstalkd` as one of the default sender

Reply:
then we must create senderMapping in event center?